interop-cats/jvm/src/main/scala/scalaz/zio/interop/catsjvm.scala

```diff
@@ -73,8 +74,8 @@ abstract class CatsInstances extends CatsInstances1 {
 
   implicit def taskEffectInstances[R](
     implicit runtime: Runtime[R]
-  ): effect.ConcurrentEffect[TaskR[R, ?]] with SemigroupK[TaskR[R, ?]] =
-    new CatsConcurrentEffect[R](runtime)
+  ): effect.ConcurrentEffect[TaskR[R, ?]] with SemigroupK[TaskR[R, ?]] with FunctionK[TaskR[R, ?], effect.IO] =
```

Comment:
Is `FunctionK` really intended to be used as an `implicit`? This would not be true for, e.g., Scalaz's `NaturalTransformation`.

Reply:
Removed